proto/proto-backplane-grpc/src/main/proto/deephaven_core/proto/table.proto

```diff
@@ -737,11 +737,22 @@ message CrossJoinTablesRequest {
 }
 
 message NaturalJoinTablesRequest {
+  enum JoinType {
+     // Throw an error if a duplicate right hand table row is found. This is the default behavior if not specified.
+    ERROR_ON_DUPLICATE = 0;
```

Comment:
Please leave 0 for "NOT SET" to catch clients that forget to assign this, rather than confuse them for having set a real value. Additionally it lets us change the default if we decided it is needed, without breaking clients that explicitly wanted "give me an error if there is a dup".

https://protobuf.dev/best-practices/dos-donts/#unspecified-enum